src/app_engine/constants.py

```diff
@@ -42,6 +42,21 @@ class EventType(object):
   # Event signifying that a room enters the state of having exactly
   # two participants.
   ROOM_SIZE_2 = 'room_size_2'
+  ICE_CONNECTION_STATE_CONNECTED = 'ice_connection_state_connected'
+
+
+class AnalyticsField(object):
```

Comment:
Are you planning to add methods here, or do these exist just for namespacing? If the latter, can these be constants instead?
Also, can this be placed in analytics.py instead?